src/components/AccountContainer.js

```diff
@@ -42,20 +42,61 @@ class AccountContainer extends Component {
           category: "Coffee",
           amount: 365
         }
-      ]
+      ],
+      filteredTransactions: [{
```

Comment:
We discussed this in lecture, but I prefer to keep only one list of transactions on the state. That way, you can filter what it being displayed instead of having to maintain this twice.